src/zui/ZUIOrganizeSidebar/index.tsx

```diff
@@ -271,137 +272,46 @@ const ZUIOrganizeSidebar = (): JSX.Element => {
                 mx: 1,
               }}
             >
-              {menuItemsMap.map((item) => {
-                const selected = key.startsWith('/' + item.name);
-                const icon = cloneElement<IconProps>(item.icon, {
-                  // Differentiate size of icon for open/closed states
-                  fontSize: open ? 'small' : 'medium',
-                });
-
+              <SearchDialog
+                activator={(openDialog) => (
+                  <Tooltip
+                    placement="right"
+                    title={
+                      !open ? messages.organizeSidebar['search']() : undefined
+                    }
+                  >
+                    <SidebarListItem
+                      data-testid="SearchDialog-activator"
+                      icon={<Search />}
+                      name="search"
+                      onClick={openDialog}
+                      open={open}
+                    />
+                  </Tooltip>
+                )}
+              />
+              {menuItemsMap.map(({ name, icon }) => {
                 return (
-                  <>
-                    {/* Add tooltip if menu is collapsed */}
-                    {!open && (
-                      <NextLink
-                        key={item.name}
-                        href={`/organize/${orgId}/${item.name}`}
-                        passHref
-                      >
-                        <Tooltip
-                          placement="right"
-                          title={messages.organizeSidebar[item.name]()}
-                        >
```

Comment:
This code was duplicated twice, the only difference was that one was wrapped in a tooltip. I moved the repeated code out to its own component and made the tooltip logic so that it only displays when closed.

Reply:
Great initiative! This means @alexandrakoch no longer needs to do that. 😊